### DISCUSSION TITLE
Calilantern calibration by Vector 3D

### DISCUSSION THREAD

**Discussion by Arnaud3767** [xblax/flashforge_ad5m_klipper_mod#257] [General]
Hello,
I wonder if buying the Vector 3D calilantern would be helpful to improve the accuracy of the printer. 
Has anyone tried this calibration tool yet?
Are the settings easily editable in Klipper?
thanks.

**Comment by jokesters** [comment]
I've used it, it improves the print quality. Especially if you do functional prints that have parts that need to fit together. I would highly recommend it. I think it is a good investment.

**Reply by Arnaud3767** [reply]
Thank you for your feedback
I will try califlower

**Reply by jtenniswood** [reply]
I use v1, it's good for dimensional accuracy, worth buying (for the right price)
